### ISSUE TITLE
Can you autoboot with Clover?

### ISSUE THREAD

**Issue by redirectto** [ryanrudolfoba/SteamDeck-Clover-dualboot#34]
In my setup I have Windows on an External device, I have two questions about autoboot.
Can you configure Clover in order to autoboot to SteamOS when holding a key at start and autoboot to Windows when holding another key at start?
Can you autoboot on SteamOS if Clover does not detect the external device with Windows and autoboot on Windows when it does?

**Comment by ryanrudolfoba** [comment]
> Can you configure Clover in order to autoboot to SteamOS when holding a key at start and autoboot to Windows when holding another key at start?

Not possible

> Can you autoboot on SteamOS if Clover does not detect the external device with Windows and autoboot on Windows when it does?

You can configure the BIOS in a similar way -

Go to BIOS > Boot > Add Boot Options. Change it from Auto to First.
What that means is if there is a USB device plugged in it will boot from it first.

Then go to Desktop Mode and launch Clover Toolbox. Select Boot > SteamOS.
What this means is when Clover starts up it will automatically set SteamOS as the default boot.

Once this is done, the behavior will be as follows -
When USB device that contains Windows is plugged in, it will automatically boot to Windows.
When USB device is disconnected, it will launch Clover and then automatically boot SteamOS.

Let me know if that helps.

**Comment by ryanrudolfoba** [comment]
havent heard back so i'll be closing this. feel free to re-open if need further help.

**Comment by redirectto** [comment]
Sorry for not answering soon enough, I had some trouble trying to install windows without another windows with administrator rights, I had to ask a coworker to help me install it.

I have a 500GB Internal SSD with SteamOS and an 1TB external SSD with Windows 11 on the go and I didn't knew if I needed a dual boot setup but it seems that it is not mandatory.

I followed your steps and they worked but at the time SteamOS had to wait for 2 things for the Steam Deck to look for external devices (due to Boot Options First) and then to load Clover and then launch SteamOS.
I tried to remove Clover and now SteamOS skips the clover launching so it is closer to what I wanted.
I would really want to have something similar to [https://github.com/derrekr/fastboot3DS](url)

Maybe in the future I'll try Clover again (or another dual boot that supports external devices) but without Boot Options First.

Thank you for your help.
